schema.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+const rootSchema = `
+type Query {
+		arrivals (stop: Int!): [Arrival]!
+		vehicle (num: Int!): Vehicle!
+		route (route: String!): [Route]!
+		headsign (headsign: String!): [Route]!
+}
+type Arrival {
+		headsign:  String!
+		stoptime:  String!
+		vehicle:   String!
```

Comment:
We can turn this into a vehicle object instead

Reply:
Researched this: While this would be a better practice schema-wise, this wouldn't be optimal as every arrival object would have to make a vehicle call in order to create the object. 

We have no store or context to tap into for this.